### ISSUE TITLE
how to do powered by google?

### ISSUE THREAD

**Issue by nyousefzai** [Tintef/react-google-places-autocomplete#147]
How we can activate powered by google, is this option been build yet?
It required by google to have this option. 
`https://developers.google.com/places/web-service/policies#powered`

**Comment by Tintef** [comment]
@nyousefzai No, it's not possible right now, but they provide you with the image, so you can add it yourself in the meantime.

A PR adding this feature is welcomed.

**Comment by nyousefzai** [comment]
sure as soon as I get a chance, at the main time here is little work around might be useful for someone else.
    `import { components } from "react-select";`

 `const Menu = (props: any) => {
  return (
    <>
      <div>Custom Menu with options</div>
      <components.Menu {...props}>
        {props.children}
        <div>img tag</div>
      </components.Menu>
    </>
  );
};
`

 `<GooglePlacesAutocomplete
          selectProps={{
            components: {
              Menu: Menu,
            },
          }}
        />`

**Comment by farrukhrashid1997** [comment]
Hi @Tintef, is this issue still open? Can I pick this up?

**Comment by Tintef** [comment]
@farrukhrashid1997 Yeah, feel free to open a PR
